File: AD-HOC/3126/main.c
#include <stdio.h>

int main() {
    int n, attended, numAttended = 0;
    scanf("%d", &n);
    for (int times = 0; times < n; times++) {
        scanf("%d", &attended);
        numAttended += attended;
    }
    printf("%d\n", numAttended);
    return 0;
}

File: AD-HOC/3126/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int numAttended = 0;
        int n = scanner.nextInt();
        for (int times = 0; times < n; times++)
            numAttended += scanner.nextInt();
        System.out.println(numAttended);
        scanner.close();
    }
}
